drizzle-orm/src/pg-core/query-builders/select.ts

```diff
@@ -203,14 +209,51 @@ export abstract class PgSelectQueryBuilder<
 		};
 	}
 
+        /**
+          * For each row of the table, include
+          * values from a matching row of the joined
+          * table, if there is a matching row. If not,
+          * all of the columns of the joined table
+          * will be set to null.
+          */
 	leftJoin = this.createJoin('left');
 
+        /**
+          * Includes all of the rows of the joined table.
+          * If there is no matching row in the main table,
+          * all the columns of the main table will be
+          * set to null.
+          */
 	rightJoin = this.createJoin('right');
 
+        /**
+          * This is the default type of join.
+          *
+          * For each row of the table, the joined table
+          * needs to have a matching row, or it will
+          * be excluded from results.
+          */
 	innerJoin = this.createJoin('inner');
 
+        /**
+          * Rows from both the main & joined are included,
+          * regardless of whether or not they have matching
+          * rows in the other table.
+          */
 	fullJoin = this.createJoin('full');
 
+        /**
```

Comment:
This uses spaces instead of tabs - please format the files you changed with Dprint.

Reply:
Okay - ran dprint over them.